src/vs/workbench/contrib/terminal/electron-sandbox/localTerminalService.ts

```diff
@@ -125,7 +125,7 @@ export class LocalTerminalService extends Disposable implements ILocalTerminalSe
 			workspaceId: this._getWorkspaceId(),
 			tabs: layoutInfo ? layoutInfo.tabs : []
 		};
-		await this._localPtyService.setTerminalLayoutInfo(args);
+		await this._localPtyService?.setTerminalLayoutInfo(args);
```

Comment:
This isn't that safe since `_localPtyService` is not `| undefined` to begin with. I wouldn't have thought `LocalTerminalService` would be getting instantiated to begin with in web since it lives in `electron-sandbox`?